src/CommunityToolkit.Maui/HandlerImpl/Popup/BasePopup.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Core.Handlers;
+using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Views;
+public partial class Popup
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	public static CommandMapper<IPopup, PopupViewHandler> ControlPopUpCommandMapper = new(PopupViewHandler.PopUpCommandMapper)
+	{
+#if IOS
+		[nameof(IPopup.OnOpened)] = MapOnOpened
+#endif
+	};
+
+	internal static void RemapForControls()
+	{
+		PopupViewHandler.PopUpCommandMapper = ControlPopUpCommandMapper;
```

Comment:
This overwrites the original `ControlPopupCommandMapper` assignment. 

Is this intended? No worries if it is, but just want to make sure!

```cs
	public static CommandMapper<IPopup, PopupViewHandler> PopUpCommandMapper = new(ElementCommandMapper)
	{
		[nameof(IPopup.OnDismissed)] = MapOnDismissed,
#if !(IOS || MACCATALYST)
		[nameof(IPopup.OnOpened)] = MapOnOpened,
#endif
		[nameof(IPopup.LightDismiss)] = MapOnLightDismiss
	};
```

Reply:
Yes, it's the same approach they use [here](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/HandlerImpl/Label/Label.cs). We can use this _movement_ when we need to implement something on Handlers that needs the `Maui. Controls` namespace